index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Amol Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <nav class="nav-bar">
+            <div class="ham-menu">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+        </div>
+        </nav>
+    </header>
+    <main class="Backgroud">
+       <section >
+        <div class="title-name">
+            <h1>I’m Amol<br>
+                Glad to see you!
+            </h1>
+        </div>
+        <p class="title-intro">
+            I’m a software developer! I can help you build a product , feature or website
+                 Look through some of my work and experience! If you like what you see 
+                 and have a project you need coded, don’t hestiate to contact me.
+            
+        </p>
+        <div class="Connect" >
+            <a class="dontwrap">LET'S CONNECT</a>
+        </div>
+        <div  class="Social-Icon">
+            <ul>
+                <li> <a href = "#" > <img src= "image/twitter.png" alt="twitter" class="twiiter-symbol"> </a> </li>
+                <li> <a href = "#" > <img src= "image/i.png"  alt="Linkedin" class="linkedIn-Isymbol"> </a> </li>
+                <li> <a href = "#" > <img src= "image/m.png" alt="M" class="symbol-m"> </a> </li>
+                <li> <a href = "#" > <img src= "image/cat.png" alt="cat" class="Cat-Symbol"> </a> </li>
```

Comment:
As I noticed after you have changed the comments of the previous reviewer the image `src` for these three icons is not functioning correctly. Kindly, make sure to rename the names of the images since it's case-sensitive rename them to the upper case I see you have upper case names in your image folder.